ODIN_II/SRC/parse_making_ast.cpp

```diff
@@ -1860,12 +1868,12 @@ void graphVizOutputAst_traverse_node(FILE* fp, ast_node_t* node, ast_node_t* fro
             if (node->types.variable.is_reg) temp << " REG";
 
             fprintf(fp, ": %s", temp.str().c_str());
+            skip_children = true;
             break;
         }
         case NUMBERS:
-            fprintf(fp, ": %s (%s)",
-                    node->types.vnumber->to_vstring('h').c_str(),
-                    node->types.vnumber->to_vstring('s').c_str());
+            fprintf(fp, " %ld",
```

Comment:
I would prefer if you kept the bit string since some numbers could be of 64 bit which don't fit in a long